package.json

```diff
@@ -88,6 +88,7 @@
   },
   "dependencies": {
     "auth0-js": "^9.7.3",
+    "auth0-password-policies": "auth0/auth0-password-policies#v1.0.1",
```

Comment:
I'd trust more on password-sheriff than this repo. Sheriff already defines them here: https://github.com/auth0/password-sheriff/blob/4a17c9cba614ed8b75affc7c53ff0d4ba31a5b30/index.js. If this is not possible, then please copy that file and keep it here on this repo.

Reply:
our change password widget also uses this repo. This is fine